javascript-forms/form-events.js

```diff
@@ -0,0 +1,19 @@
+var $focus = document.querySelector('#user-name');
+console.log($focus);
+$focus.addEventListener('focus', function (event) {
```

Comment:
there should be a `focus` `blur` and `input` function for each input. 9 in total